src/NzbDrone.Common/Http/HttpRequestBuilder.cs

```diff
@@ -296,7 +296,8 @@ public virtual HttpRequestBuilder AddQueryParam(string key, object value, bool r
                 SuffixQueryParams.RemoveAll(v => v.Key == key);
             }
 
-            QueryParams.Add(key, value.ToString());
+            var valueStr = value?.ToString() ?? "";
```

Comment:
Can you work around the limitations of HttpRequestBuilder? I would like to avoid changing anything in here to make merging commits from Sonarr easier and less error prone.

Reply:
Hmm, I could definitely do without this, but I also think setting the request method is a pretty basic part of an HTTP request builder. I suppose I'll do it here without, and see if this gets pulled upstream, i.e. Sonarr.
I didn't know thought, that Radarr still pulls changes from Sonarr. Interesting